Makefile

```diff
@@ -342,5 +343,9 @@ undeploy-olm: ## Un-deploy the operator with OLM
 DEFAULT_OLM_NAMESPACE ?= openshift-marketplace
 .PHONY: catalog-update
 catalog-update: ## Update catalog source in the default namespace for catalogsource
-	-kubectl delete catalogsource backstage-operator -n $(DEFAULT_OLM_NAMESPACE)
+	kubectl delete catalogsource backstage-operator -n $(DEFAULT_OLM_NAMESPACE)
 	sed "s/{{CATALOG_IMG}}/$(subst /,\/,$(CATALOG_IMG))/g" config/samples/catalog-source-template.yaml | kubectl apply -n $(DEFAULT_OLM_NAMESPACE) -f -
+
+.PHONY: deploy-openshift
```

Comment:
Why do we need this? We already have deploy-olm. Also this only deploys the catalog source, not the operator subscription, and "deploy-openshift" is confusing to me.

Reply:
That's how me and @rm3l  (as I understand) deploy it to Openshift to test. 
Do you see any problem?